src/main/java/com/uber/cadence/internal/replay/QueryWorkflowParameters.java

```diff
@@ -45,11 +45,6 @@ public void setInput(byte[] input) {
     this.input = input;
   }
 
-  public QueryWorkflowParameters withInput(byte[] input) {
```

Comment:
I don't feel strongly, but isn't the `withX` pattern preferable to setters because it's chainable?

Reply:
right but they are currently unused though